### PR TITLE
Skip loading Jetpack for FedRAMP sites by default

### PR DESCRIPTION
**Pull request by jblz**
## Description

In the `000-vip-init.php` file, when the `VIP_IS_FEDRAMP` constant is true, set the `VIP_JETPACK_SKIP_LOAD` constant to `true` as well.

## Checklist

Please make sure the items below have been covered before requesting a review:

- [x] This change works and has been tested locally (or has an appropriate fallback).
- [ ] This change works and has been tested on a Go sandbox.
- [ ] This change has relevant unit tests (if applicable).
- [ ] This change uses a rollout method to ease with deployment (if applicable - especially for large scale actions that require writes).
- [ ] This change has relevant documentation additions / updates (if applicable).
- [x] I've created a changelog description that aligns with the provided examples.

## Steps to Test

1. Check out PR.
2. Test that Jetpack is enabled for non-FedRAMP sites
    * Browse to `http://$SITE_URL/index.php?rest_route=/jetpack/v4`
        * You should see a list of Jetpack REST API endpoints
    * Browse to `wp-admin` while logged in
        * You should see Jetpack settings in the menu & the options page should function as expected
3. Test that Jetpack is *NOT* enabled for FedRAMP sites
    * Add the following to your `vip-config/vip-config.php` file:
    `define( 'VIP_IS_FEDRAMP', true );`
    * Browse to `http://$SITE_URL/index.php?rest_route=/jetpack/v4`
        * You should see a `404` w/ `rest_no_route` code.
    * Browse to `wp-admin` while logged in
        * You should *NOT* see Jetpack settings in the menu & manually browsing to `http://$SITE_URL/wp-admin/admin.php?page=jetpack` should `403` with a "Sorry, you are not allowed to access this page." message.

### CODE REVIEW

**Review by rinatkhaziev — CHANGES_REQUESTED** [review]
@jblz I think maybe rather than check for the constant like that we should make this a [Context](https://github.com/Automattic/vip-go-mu-plugins/blob/develop/lib/utils/class-context.php) helper? Like `Context::is_fedramp` or something.

The rationale is likely we'll re-use this check in many other parts of codebase (At least Parse.ly comes to mind)

**Comment by sjinks** [comment]
@rinatkhaziev Done, please take a look

**Review by rinatkhaziev — APPROVED** [review]

**Comment by jblz** [comment]
Thanks for the refactor @sjinks !

File: 000-vip-init.php [hunk 1]
```diff
@@ -128,6 +128,11 @@
 // Define constants for custom VIP Go paths
 define( 'WPCOM_VIP_CLIENT_MU_PLUGIN_DIR', WP_CONTENT_DIR . '/client-mu-plugins' );
 
+// FedRAMP sites do not load Jetpack by default
+if ( method_exists( Context::class, 'is_fedramp' ) && Context::is_fedramp() && ! defined( 'VIP_JETPACK_SKIP_LOAD' ) ) {
```
Comment (jblz) [inline 000-vip-init.php:132]: The class is probably as likely to be missing as the function, so I'd probably use `is_callable` instead of `method_exists`.

Comment (rinatkhaziev) [reply 000-vip-init.php:132]: No, class is definitely there. But lib/ will be copied over later than 000-vip-init

**Pull request merged**